backend/tests/test_query_count.py

```
@@ -0,0 +1,77 @@
+
+
+class QueryCountTest(TransactionTestCase):
+    """
+    Test that API endpoints don't make excessive database queries.
+    """
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects.create_user(
+            username='testuser',
+            password='testpass123',
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)